### ISSUE TITLE
Migration to Manifest v3

### ISSUE THREAD

**Issue by tobiasdiez** [zotero/zotero-connectors#372]
Google Chrome sunsets the existing manifest v2 in January 2023. At this point, extensions using v2 stop working: https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/

The biggest change with respect to how the Zotero Connector is currently working is that [remote code can no longer be executed](https://developer.chrome.com/docs/extensions/mv3/intro/mv3-overview/#remotely-hosted-code). In particular, Zotero's strategy of downloading translators on-demand doesn't work anymore and they would be needed to be bundled in the connector. 

Migration guide: https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/

**Comment by dstillman** [comment]
Yes, thanks, we're well aware. It will require fairly extensive changes across the extension, but we've been working on it for a while.

**Comment by adomasven** [comment]
Done
